hazelcast/proxy/map.py

```diff
@@ -303,9 +725,30 @@ def set(self, key, value, ttl=-1):
         return self._set_internal(key_data, value_data, ttl)
 
     def size(self):
+        """
+        Returns the number of entries in this map.
+
+        :return: (int), number of entries in this map.
+        """
         return self._encode_invoke(map_size_codec)
 
     def try_lock(self, key, ttl=-1, timeout=0):
+        """
+        Tries to acquire the lock for the specified key. When the lock is not available,
+
+            * If timeout is not provided, then the current thread doesn't wait and returns false immediately.
+            * If a timeout is provided, then the current thread becomes disabled for thread scheduling purposes and lies
```

Comment:
* If a timeout is provided, the current thread becomes disabled for thread scheduling purposes and lies...